dist/schema.json

```diff
@@ -31165,7 +31165,7 @@
           "value"
         ],
         "externalDocs": {
-          "url": "https://docs.joinmastodon.org/entities/Admin_DimensionData/",
+          "url": "https://docs.joinmastodon.org/entities/Admin_Dimension/#admin",
```

Comment:
Some URLs are not correct. `Admin::DimensionData` for example should be linking to `https://docs.joinmastodon.org/entities/Admin_Dimension/#data-attributes`.

The primary entity on each entity URL should have the fragment `#attributes`.